### PR TITLE
Adding VLHLA

### PR DESCRIPTION
**Pull request by t0m3kf**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -1787,5 +1787,20 @@
...
+    "social":{
+      "facebook":"",
+      "twitter":"https://twitter.com/valhallapool",
+      "website":"http://valhallapool.net",
+      "reddit":"",
+      "instagram":"",
+      "youtube":""
+    },
+    "telegram":"valhallapool",
+    "operatorName":"valhallapool",
```
Comment (lophre) [inline registry.json:1803]: If you don't provide a real first name or full name, please leave this field empty.

File: registry.json [hunk 2]
```diff
@@ -1787,5 +1787,20 @@
...
+    "poolId":"29889ecc8151e5a93fcd321cd905eee0f50c8364273550205058bfe8",
+    "social":{
+      "facebook":"",
+      "twitter":"https://twitter.com/valhallapool",
+      "website":"http://valhallapool.net",
+      "reddit":"",
+      "instagram":"",
+      "youtube":""
+    },
+    "telegram":"valhallapool",
```
Comment (lophre) [inline registry.json:1802]: Can't find this Telegram username

**Review by lophre — CHANGES_REQUESTED** [review]
Please see comments

**Comment by t0m3kf** [comment]
Updated telegram handle and removed operatorName value

File: registry.json (hunk 2 above)
Comment (t0m3kf) [reply registry.json]: Resolved with commit dba82500c4dd7dece29df2cc7c7813fc24a6a0ac

File: registry.json (hunk 1 above)
Comment (t0m3kf) [reply registry.json]: resolved with dba82500c4dd7dece29df2cc7c7813fc24a6a0ac

**Review by t0m3kf — COMMENTED** [review]
Changed telegram handle and removed operatorName value

**Review by lophre — APPROVED** [review]

**Review by lophre — APPROVED** [review]

**Pull request merged**